velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -200,6 +200,11 @@ class MemoryAllocatorTest : public testing::TestWithParam<TestParam> {
     MemoryAllocator::testingDestroyInstance();
   }
 
+  void reset() {
```

Comment:
I wonder if it would be cleaner to make separate tests so that 'reset' is done by the gtest itself.

Reply:
It is to test different allocation parameters. I will make a little bit refactoring so that reset() won't call gtest method directly.